cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -234,16 +247,23 @@ func main() {
 		provisionerOptions...,
 	)
 
+	csiClaimController := ctrl.NewCloningProtectionController(
+		clientset,
+		claimLister,
+		claimInformer,
+		claimQueue,
+	)
+
 	run := func(context.Context) {
-		stopCh := context.Background().Done()
```

Comment:
Why did you remove `stopCh`?

Reply:
Accidentally left this change after seeing issues with a similar approach in tests, will undo this.